doc/source/comparison_with_sas.rst

```diff
@@ -357,6 +357,142 @@ takes a list of columns to sort by.
    tips = tips.sort_values(['sex', 'total_bill'])
    tips.head()
 
```

Comment:
can you add a directive here (and other sub-sections if you can), for references e.g. ``_compare_with_sas.string``